templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -480,4 +493,23 @@ export class {{ service.name }}Client {
     return this._innerApiCalls.{{ method.name.toCamelCase() }}(request, options, callback);
   } 
 {%- endfor %}
-}
+{%- if (service.pathTemplate.length > 0) %}
+  // --------------------
+  // -- Path templates --
+  // --------------------
+{%- for template in service.pathTemplate %}
+  {{ template.type.toLowerCase() }}Path({{ template.params.addStringType() }}){
```

Comment:
I suspect it might be `toCamelCase()` here - do we have any resources with multiple words?

Reply:
I'm not sure we have multiple-words resources, but yea I think camel case is better here.